reserve_sdk/token.py

```diff
@@ -0,0 +1,18 @@
+from .utils import call_contract
+
+
+class Token():
```

Comment:
could you add a comment doc for this class? Also, should it be named ERC20 token?